src/lib/seo/page-metadata.ts

```
@@ -0,0 +1,216 @@
+import { Metadata } from 'next';
+
+export interface PageMetadataConfig {
+  title: string;
+  description: string;
+  keywords?: string[];
+  ogImage?: string;
+  canonicalUrl?: string;
+  noIndex?: boolean;
+  structuredData?: Record<string, any>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)